SampleViewer/qml/main.qml

```diff
@@ -260,17 +260,9 @@ ApplicationWindow {
             }
         }
 
-        function onApiKeyRequired(apiKey) {
-            qmlArcGISRuntimeEnvironment.setSource("qrc:/qmlsamples/QmlArcGISRuntimeEnvironment.qml", {apiKey: apiKey})
-        }
```

Comment:
Is there no equivalent for this in C++ sample viewer, or does it already exist elsewhere and this is redundant?

Reply:
this was a solution only for QML. The C++ equivalent is taken care of where this was emitted previously. See my other comment.